### DISCUSSION TITLE
How to plan a route with POIs?

### DISCUSSION THREAD

**Discussion by CornerSyrup** [osmandapp/OsmAnd#19030] [Q&A]
I noticed when I plan a route and add points, I can only add by tapping the screen. Even though I'm tapping on the POI icon, it's still using the coordinate of my tap instead of the POI.

- Is there any way I can plan with POIs?
- How to use waypoints I added to track file to plan routes?
- Is it the only way to plan routes with POI is planning with navigation and save as track? If so, how to join routes?

**Comment by ZeljkoBG** [comment]
That's an incredible omission! The possibility of adding coordinates as a route point should also be added.

PS: reading other discussions. There are no living people here :)

**Comment by DmitryAlexei** [comment]
Please try Navigation using map markers. You can read more in the article https://osmand.net/docs/user/navigation/setup/markers-navigation#overview
Map markers can be created from a group of favorites. Don't forget to update OsmAnd to the latest version.

**Reply by CornerSyrup** [reply]
I'm using this approach currently. But it seems misleading to me, since Navigation lead to navigate instead of planning, besides there's a function call "plan a route"...

If you wouldn't mind, can you tell me how to use waypoints added to track file? I can't find related description in docs... Is it just to display on the map, or it contribute to some feature (e.g. editing/creating routes)?

**Comment by ZeljkoBG** [comment]
This is what the Dialog Box for **New Routes** should look like

![IMG_20240306_181654](https://github.com/osmandapp/OsmAnd/assets/53188843/6a06c1f2-84e9-4331-a077-c16b86ce5bcf)

**Home** and **Work** should be under **Favorites**. The green frame is nothing.

-------------------------
Meni/Plan a route/Create new route - it is not clear box. Unnecessary.
